djconnectwise/api.py

```diff
@@ -189,14 +189,19 @@ def _prepare_error_response(self, response):
         # error = error.args[0].decode("utf-8")
         error = error.replace('\r\n', '')
         messages = []
+
         try:
             error = json.loads(error)
+            primary_error_msg = '{}.'.format(error.get('message'))
```

Comment:
Note that originally this was a semi colon. I replaced it with a period to be more flexible in case `messages` in line 203 is empty.